packages/vite/package.json

```
@@ -28,6 +28,7 @@
       "types": "./client.d.ts"
     },
     "./dist/client/*": "./dist/client/*",
+    "./types/*": "./types/*",
```

```suggestion
    "./types/*": {
      "types": "./types/*"
    },
```
I think we should only limit this to the types condition.

---

I have fixed. Please re-review.